examples/aws-elb/README.md

```diff
@@ -1,11 +1,11 @@
 # ELB with stickiness Example
 
-The example launches a web server, installs nginx, creates an ELB for instance. It also creates security groups for elb/instance 
+The example launches a web server, installs nginx, creates an ELB for instance. It also creates security groups for the ELB and EC2 instance. 
 
 To run, configure your AWS provider as described in https://www.terraform.io/docs/providers/aws/index.html
 
-Running the example
+Run this example using:
 
-run `terraform apply -var 'key_name={your_key_name}}'` 
+    terraform apply -var 'key_name={your_key_name}}' 
```

Comment:
I would prefer to use `YOUR_KEY_NAME` (without `{` and `}`) here. At least the second `}` should be removed.